jcl/src/java.base/share/classes/java/lang/invoke/DirectHandle.java

```diff
@@ -86,13 +86,13 @@ public DirectHandle(Method method, byte kind, Class<?> specialCaller, boolean or
 		addHandleToClassCache();
 		this.vmSlot = other.vmSlot;
 		this.defc = other.defc;
-		final_modifiers = rawModifiers;
+		isStatic = Modifier.isStatic(other.rawModifiers);
```

Comment:
Stylistically, this would read better if `isStatic` were referenced as `this.isStatic` like other fields.

Reply:
The use of `this` was to make it clear when setting identically named fields.  In this case, there's no other `isStatic` field that needs to be distinguished.

Reply:
I understand when the qualification is needed, and that it is not *needed* here which is why I described it as a stylistic issue: all fields in all constructors use `this.` except `isStatic` - I was requesting consistency.

Reply:
I find unnecessary `this.` qualifiers to add visual noise and to be unnecessary in this case.

Reply:
I agreed they were unnecessary, but, IMO, the different initialization styles is more distracting: I won't push it further, however.